en/config_vtol/vtol_without_airspeed_sensor.md

```diff
@@ -1,23 +1,25 @@
 # VTOL Without an Airspeed Sensor
 
+<Badge type="warning" text="Experimental" />

```

Comment:
FYI only, I'm experimenting with badges to indicate this kind of thing "as well". I hope to link to a definition at some point.